src/seedu/addressbook/data/menu/Price.java

```diff
@@ -29,6 +30,22 @@ public Price(String price/*, boolean isPrivate*/) throws IllegalValueException {
         this.value = price;
     }
 
+    public double convertPricetoDouble(String result){
+        result = this.value;
+        String doublevalue = result.substring(1);
+        double priceIndouble = Double.parseDouble(doublevalue);
+
+        return priceIndouble;
+    }
+
+    public String convertPricetoString(double priceIndouble){
```

Comment:
This method can be adjusted to convert the double to the correct 2 digits after the dot format